packages/design-carousel/src/index.tsx

```diff
@@ -1,9 +1,12 @@
-import { useStarterDesignsQuery } from '@automattic/data-stores';
+import { Gridicon } from '@automattic/components';
+import { StarterDesigns, useStarterDesignsQuery } from '@automattic/data-stores';
 import { useLocale } from '@automattic/i18n-utils';
+import { MShotsOptions } from '@automattic/onboarding';
```

Comment:
Should this not be a type import?
```suggestion
import type { MShotsOptions } from '@automattic/onboarding';
```